lib/components/Nav.jsx

```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
     <nav
       className='justify-end items-center hidden sm:flex w-2/3'
     >
+      <Link
+        href='https://www.notion.so/PoolTogether-Knowledge-Base-fa721ccefa3242eaabd125a8415acd27'
+        as='https://www.notion.so/PoolTogether-Knowledge-Base-fa721ccefa3242eaabd125a8415acd27'
+        shallow
+      >
+        <a
+          target='_blank'
+          className={classnames(
+            'mr-3',
+            navLinkClasses,
+            {
+              'text-white hover:text-highlight-2': !developersPage,
+              'text-highlight-2 hover:text-highlight-2': developersPage
```

Comment:
These class differences only apply to internal links (ie. links on https://pooltogether.com). If we're linking out to an external site (notion in this case) we can simply use these two classes: `text-white hover:text-highlight-2`